tests/unit/test_app.py

```diff
@@ -8,6 +8,9 @@
 from chalice import app
 from chalice import NotFoundError
 from chalice import __version__ as chalice_version
+from tests.unit.conftest import create_event
```

Comment:
I don't think conftest is meant to be imported directly (or at least it's something I see).  I'd suggest converting them to fixtures if you want to move them there.

I think I might have missed this in the wheel based packager PR.